edgeai-apps-utils/suite/bookworm/debian/rules

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/make -f
+
+export SOC=am62x
+
+%:
+	dh $@
+
+override_dh_auto_configure:
+	mkdir build/ && cd build/ && cmake ..
```

Comment:
Current debhelper should be able to build cmake projects automatically, these overrides shouldn't be needed.